packages/web-twig/src/Resources/components/CheckboxField/CheckboxField.twig

```diff
@@ -50,14 +50,14 @@
         id="{{ _id }}"
         name="{{ _name }}"
         class="{{ _inputClassName }}"
-        {{ _checkedAttr | raw }}
-        {{ _disabledAttr | raw }}
-        {{ _requiredAttr | raw }}
-        {{ _valueAttr | raw }}
+        {{ _checkedAttr  }}
+        {{ _disabledAttr  }}
+        {{ _requiredAttr  }}
+        {{ _valueAttr  }}
     />
     <span class="{{ _textClassName }}">
-        <span {{ classProp(_labelClassName) }}>{{ _label | raw }}</span>
-        {% if _helperText %}<span class="{{ _helperTextClassName }}">{{ _helperText | raw }}</span>{% endif %}
-        {% if _message %}<span class="{{ _messageClassName }}">{{ _message | raw }}</span>{% endif %}
+        <span {{ classProp(_labelClassName) }}>{{ _label  }}</span>
```

Comment:
⚠️ The checkbox label in some cases contains link so we need apply `raw` filter.

I'm not sure what about `helperText` or `message` (even the design guidelines probably don't allowed it, it's possible that somewhere props pass formatted markup).

Reply:
[Typo in tests.](https://github.com/lmc-eu/spirit-design-system/blob/49d5f7e18747a2884de9f736a55eb94dcb489c94/packages/web-twig/tests/components-fixtures/checkboxFieldDefault.twig#L1) → Could you please fix prop name `messsage` → `message`? Looks like it was copypasted across form fields. 🙈

Reply:
> ⚠️ The checkbox label in some cases contains link so we need apply `raw` filter.
> 
> I'm not sure what about `helperText` or `message` (even the design guidelines probably don't allowed it, it's possible that somewhere props pass formatted markup).

In a future implementation, I think the label should be a component.

Reply:
Typo was fixed in separate PR https://github.com/lmc-eu/spirit-design-system/pull/743

Reply:
For `label`, `message` and the `helperText` I have introduced `UNSAFE_` variants that are using `raw` filter. Variants without the prefix are from now escaped.